Sources/SWBUniversalPlatform/Specs/Swift.xcspec

```diff
@@ -806,6 +806,32 @@
                 Description = "Changes existential types to require explicit annotation with the `any` keyword.";
             },
 
+            {
+                Name = "SWIFT_UPCOMING_FEATURE_INFER_ISOLATED_CONFORMANCES";
+                Type = Boolean;
+                DefaultValue = NO;
+                CommandLineArgs = {
+                    YES = ( "-enable-upcoming-feature", "InferIsolatedConformances" );
+                    NO  = ();
+                };
+                DisplayName = "Infer Isolated Conformances";
+                Category = "Upcoming Features";
+                Description = "Infer conformances of global-actor isolated types as isolated to the same actor unless isolation is explicitly specified as `nonisolated`.";
+            },
+
+            {
+                Name = "SWIFT_UPCOMING_FEATURE_NONISOLATED_NONSENDING_BY_DEFAULT";
+                Type = Boolean;
+                DefaultValue = NO;
+                CommandLineArgs = {
+                    YES = ( "-enable-upcoming-feature", "NonisolatedNonsendingByDefault" );
+                    NO  = ();
+                };
+                DisplayName = "Nonisolated Nonsending By Default";
```

Comment:
Should Nonsending be Non-sending? Not sure how we've been spelling this elsewhere